docs/source/getting-started/tutorial.rst

```diff
@@ -1,6 +1,6 @@
 .. _ai-training:
 
-Tutorial: AI Training
+Quickstart: PyTorch
```

Comment:
Can we add the included files section in this page as well, i.e. files in https://github.com/skypilot-org/skypilot/tree/master/examples/distributed-pytorch

Reply:
The content of this page + YAML are different than https://github.com/skypilot-org/skypilot/tree/master/examples/distributed-pytorch